src/wp-includes/block-editor.php

```diff
@@ -844,5 +845,10 @@ function get_classic_theme_supports_block_editor_settings() {
 		$theme_settings['gradients'] = $gradient_presets;
 	}
 
+	$spacing_sizes = current( (array) get_theme_support( 'editor-spacing-sizes' ) );
+	if ( false !== $spacing_sizes ) {
+		$theme_settings['spacingSizes'] = $spacing_sizes;
+	}
```

Comment:
This file didn't exist in gutenberg, but was required for:

- https://github.com/WordPress/gutenberg/pull/62252